group.yml

```diff
@@ -204,3 +204,5 @@ scan_freshness:
   release_regex: '(?x) ^ (\d\d\d\d) (\d\d) (\d\d) (\d\d)'
   threshold_hours: 6
 csv_namespace: openshift
+# whether and how to check if all buildroots have consistent versions of golang compilers (rpm build only)
+check_golang_versions: x.y  # "x.y" (default): only major and minor version; "exact": the z-version must be the same; "no": do not check
```

Comment:
nice. yes, i think we'd like `exact` here, except when we're stuck between devtools builds (they don't start on el7 builds until they're done w/ el8 builds) on master.